packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -86,11 +89,11 @@ const TradersHub = () => {
                     <DesktopWrapper>
                         <div className='traders-hub__main-container'>
                             <OptionsAndMultipliersListing />
-                            <CFDsListing />
+                            {is_landing_company_loaded && isMT5Allowed(landing_company) && <CFDsListing />}
                         </div>
                     </DesktopWrapper>
                     <MobileWrapper>
-                        {is_landing_company_loaded ? (
+                        {is_landing_company_loaded && isMT5Allowed(landing_company) ? (
```

Comment:
we need to show this  `ButtonToggle` even if we do not have CFDList, the thing is to hide cfd part when `!isMT5Allowed(landing_company)` in line 108

Reply:
Resolved issue 🙏🏼